### ISSUE TITLE
font_indice font_indice 

### ISSUE THREAD

**Issue by jingmingtao** [clovaai/mxfont#30]
Excuse me, Where  are font_indice, font_indice  parameter passed from discriminator.py and what does it mean

**Comment by jingmingtao** [comment]
class CustomDiscriminator(nn.Module):
    """
    spectral norm + ResBlock + Multi-task Discriminator (No patchGAN)
    """
    def __init__(self, feats, gap, projD):
        super().__init__()
        self.feats = feats
        self.gap = gap
        self.projD = projD
 def forward(self, x, font_indice, char_indice, out_feats='none'):
        assert out_feats in {'none', 'all'}
        feats = []
        for layer in self.feats:
            x = layer(x)
            feats.append(x)

        x = self.gap(x)  # final features
        ret = self.projD(x, font_indice, char_indice)

**Comment by 8uos** [comment]
It is the label of font style, which is used for computing the adversarial loss.
You can find the detailed explanation of the adversarial loss in Appendix B.4 of our paper.

**Comment by jingmingtao** [comment]
thanks very much ! another question:The training set consists of content and style, with 6763 characters for content and 556 characters for style. May I ask if this dataset combination is reasonable
